examples/AdminClient/Program.cs

```diff
@@ -93,10 +289,11 @@ public static async Task Main(string[] args)
         {
             if (args.Length < 2)
             {
-                Console.WriteLine("usage: .. <bootstrapServers> <list-groups|metadata|library-version|create-topic> [topic-name]");
+                Console.WriteLine("usage: .. <bootstrapServers> <list-groups|metadata|library-version|create-topic|create-acls|describe-acls|delete-acls> ..");
                 System.Environment.Exit(1);
             }
-            
+
+            var commandArgs = args.Skip(2).ToArray();
```

Comment:
maybe introduce a variable for bootstrapServers here, and don't use args[0] below.